types/auth0-js/index.d.ts

```diff
@@ -574,6 +574,11 @@ export interface AuthOptions {
     __tryLocalStorageFirst?: boolean;
 }
 
+export interface Captcha {
+    reload: (done: any) => void;
```

Comment:
this one should be probably:
```ts
type DoneCallback = (err?: Error) => void;
...
reload: (done?: DoneCallback) => void;
```
https://auth0.github.io/auth0.js/web-auth_captcha.js.html

Reply:
Good catch, thanks! 

I feel the error type should probably be `Auth0Error` because `done(err)` is called by `auth0Client.getChallenge(...)` which executes `wrapCallback` of helper/response-handler.js. And the error object there seems to have the same properties as Auth0Error. 

https://github.com/auth0/auth0.js/blob/master/src/helper/response-handler.js#L84

Do you agree @peterblazejewicz ?

Reply:
Oh, yes, sorry, it was just quick/dirty sample code

Reply:
Yeah, no worries! Just asked to be sure. :)